lib/features/composer/presentation/composer_controller.dart

```diff
@@ -862,134 +865,128 @@ class ComposerController extends BaseController {
     _handleSendMessages(context);
   }
 
-  void _handleSendMessages(BuildContext context) async {
-    final session = mailboxDashBoardController.sessionCurrent;
-    final arguments = composerArguments.value;
-    final accountId = mailboxDashBoardController.accountId.value;
-    final sentMailboxId = mailboxDashBoardController.mapDefaultMailboxIdByRole[PresentationMailbox.roleSent];
-    final outboxMailboxId = mailboxDashBoardController.outboxMailbox?.id;
-    final userProfile = mailboxDashBoardController.userProfile.value;
+  bool get _isParamUserNull {
+    if (composerArguments.value == null ||
+        mailboxDashBoardController.userProfile.value == null ||
+        mailboxDashBoardController.sessionCurrent == null ||
+        mailboxDashBoardController.accountId.value == null
+    ) {
+      logError('ComposerController::isParamUserNotNull: Param is NULL');
+      return true;
+    }
+    return false;
+  }
```

Comment:
```suggestion
  void _validateUserParams {
    if (composerArguments.value == null ||
        mailboxDashBoardController.userProfile.value == null ||
        mailboxDashBoardController.sessionCurrent == null ||
        mailboxDashBoardController.accountId.value == null
    ) {
      throw `ABC Exception`;
    }
    return;
  }
```
then handle exception in `final`

Reply:
We don't take any action on it. IMO, it is not necessary to do so on the presentation layer